File: app/src/main/java/com/user/canopas/dagger/mvp/model/Actors.java
package com.user.canopas.dagger.mvp.model;


public class Actors {
    public String country;
    public String image;
    public String children;
    public String dob;
    public String name;
    public String description;
    public String spouse;
    public String height;

    public Actors(String name, String image, String country, String children, String dob, String description, String spouse, String height) {
        this.name = name;
        this.image = image;
        this.children = children;
        this.country = country;
        this.dob = dob;
        this.description = description;
        this.spouse = spouse;
        this.height = height;

    }

    public Actors() {

    }

    public String getCountry() {
        return this.country;
    }


    public String getImage() {
        return this.image;
    }


    public String getChildren() {
        return this.children;
    }


    public String getDob() {
        return this.dob;
    }


    public String getName() {
        return this.name;
    }


    public String getDescription() {
        return this.description;
    }


    public String getSpouse() {
        return this.spouse;
    }

    public String getHeight() {
        return this.height;
    }

}

File: app/src/main/java/com/user/canopas/dagger/mvp/model/Actors.kt
package com.user.canopas.dagger.mvp.model


class Actors {
   lateinit var country: String
    lateinit var image: String
    lateinit var children: String
    lateinit var dob: String
    lateinit var name: String
    lateinit var description: String
    lateinit var spouse: String
    lateinit var height: String

    constructor(name: String, image: String, country: String, children: String, dob: String, description: String, spouse: String, height: String) {
        this.name = name
        this.image = image
        this.children = children
        this.country = country
        this.dob = dob
        this.description = description
        this.spouse = spouse
        this.height = height

    }

    constructor() {

    }

}
